inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -606,69 +605,67 @@ struct jit_uni_mvn_kernel_f32 : public jit_uni_mvn_kernel, public jit_generator
-    if (!descs.empty())
-        return;
-
+        : MKLDNNNode(op, eng, cache) {
     std::string errPrefix = "MVN node with name '" + getName() + "' ";
 
-    auto cnnLayer = getCnnLayer();
-    if (cnnLayer == nullptr)
-        THROW_IE_EXCEPTION << errPrefix << "does not have CNN layer.";
-
-    if (getParentEdges().size() > 2)
-        THROW_IE_EXCEPTION << errPrefix << "has incorrect number of input edges.";
-
-    if (getChildEdges().empty())
+    if (op->get_output_size() == 0)
         THROW_IE_EXCEPTION << errPrefix << "has incorrect number of output edges.";
 
-    const auto& numOfDims = getParentEdgeAt(0)->getDims().ndims();
-    if (numOfDims < 1 || numOfDims > 5)
-        THROW_IE_EXCEPTION << errPrefix << "doesn't support input with size of dimensions: " << numOfDims;
-
-    across_channels = false;
-    if (getParentEdges().size() == 1) {
-        across_channels = cnnLayer->GetParamAsBool("across_channels");
+    const auto& inDataShapeSize = op->input_value(0).get_shape().size();
+    if (inDataShapeSize < 1 || inDataShapeSize > 5)
+        THROW_IE_EXCEPTION << errPrefix << "doesn't support input with size of dimensions: " << inDataShapeSize;
+
+    if (auto mvnOp = ngraph::as_type_ptr<ngraph::op::v6::MVN>(op)) {
+        if (mvnOp->get_input_size() > 2)
+            THROW_IE_EXCEPTION << errPrefix << "has incorrect number of input edges.";
+
+        normalizeVariance_ = mvnOp->get_normalize_variance();
+        epsValue_ = mvnOp->get_eps();
+        epsMode_ = mvnOp->get_eps_mode();
+
+        acrossChannels_ = false;
+        if (inDataShapeSize == mvnOp->input_value(1).get_shape()[0] + 1 || inDataShapeSize == 1)
+            acrossChannels_ = true;
+    } else if (auto mvnOp = ngraph::as_type_ptr<ngraph::op::v0::MVN>(op)) {
+        if (mvnOp->get_input_size() > 1)
+            THROW_IE_EXCEPTION << errPrefix << "has incorrect number of input edges.";
+
+        normalizeVariance_ = mvnOp->get_normalize_variance();
+        epsValue_ = mvnOp->get_eps();
+        epsMode_ = MVNEpsMode::INSIDE_SQRT;
+        acrossChannels_ = mvnOp->get_across_channels();
     } else {
-        if (numOfDims == getParentEdgeAt(1)->getDims().size() + 1 || numOfDims == 1)
-            across_channels = true;
-    }
-    normalize_variance = cnnLayer->GetParamAsBool("normalize_variance", true);
-    eps = cnnLayer->GetParamAsFloat("eps");
-    auto epsMode = cnnLayer->GetParamAsString("eps_mode", "");
-    if (details::CaselessEq<std::string>()(epsMode, "inside_sqrt")) {
-        epsMode_ = insideSqrt;
-    } else if (details::CaselessEq<std::string>()(epsMode, "outside_sqrt")) {
-        epsMode_ = outsideSqrt;
+        THROW_IE_EXCEPTION_WITH_STATUS(NOT_IMPLEMENTED)
+                << "CPU MVN node doesn't support ngraph operation '" << op->get_type_name() << "' with name '" << op->get_friendly_name() << "'";
     }
 }
 
+void MKLDNNMVNNode::getSupportedDescriptors() {
+    if (!descs.empty())
```

Comment:
Seems like this condition is redundant. Please remove

Reply:
Fixed